example/README.md

```diff
@@ -0,0 +1,51 @@
+Example Webpack setup
+======================
+
+Here you'll find an example Webpack configuration. The `webpack.config.js` here will output two bundles here: `application.js` and `vendor.js`, and will process an image.
```

Comment:
Maybe rephrase the whole thing a bit? E.g.:

Here we give an example Webpack configuration. The given `webpack.config.js` configuration file will output two bundles (`application.js` and `vendor.js`), as well as process images.